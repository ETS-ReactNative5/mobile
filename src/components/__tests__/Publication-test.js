import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Publication from '../Publication'

const publication = {
  citation: 'Nice citation'
}

it('renders correctly', () => {
  const tree = renderer.create(
    <Publication publication={publication} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
