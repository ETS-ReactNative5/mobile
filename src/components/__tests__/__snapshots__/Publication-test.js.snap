exports[`test renders correctly 1`] = `
<View
  style={undefined}>
  <TouchableOpacity
    activeOpacity={0.5}
    onPress={[Function]}>
    <View
      style={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            undefined,
            undefined,
          ]
        }>
        Nice citation
      </Text>
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": undefined,
              "fontFamily": "FontAwesome",
              "fontSize": 12,
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            undefined,
          ]
        }>
        ï„…
      </Text>
    </View>
  </TouchableOpacity>
</View>
`;
